src/lib/deviceConfigurationHelper.js

```diff
@@ -73,10 +83,16 @@ export const updateDirectoriesExclusions = ({
     foldersToExclude.length > 0 &&
       client
         .collection(FILES_DOCTYPE)
-        .addNotSynchronizedDirectories(deviceToConfigure, foldersToExclude),
+        .addNotSynchronizedDirectories(
+          toCozyOAuthClient(device),
+          foldersToExclude.map(toCozyDirectory)
+        ),
     foldersToInclude.length > 0 &&
       client
         .collection(FILES_DOCTYPE)
-        .removeNotSynchronizedDirectories(deviceToConfigure, foldersToInclude)
+        .removeNotSynchronizedDirectories(
+          toCozyOAuthClient(device),
+          foldersToInclude.map(toCozyDirectory)
+        )
```

Comment:
let's test this feature addNotSynchronizedDirectories and removeNotSynchronizedDirectories